CalibPPS/TimingCalibration/plugins/PPSTimingCalibrationPCLWorker.cc

```diff
@@ -85,9 +94,10 @@ void PPSTimingCalibrationPCLWorker::bookHistograms(DQMStore::IBooker& iBooker,
 void PPSTimingCalibrationPCLWorker::dqmAnalyze(const edm::Event& iEvent,
                                                const edm::EventSetup& iSetup,
                                                const TimingCalibrationHistograms& iHists) const {
-  // then extract the rechits information for later processing
   edm::Handle<edm::DetSetVector<CTPPSDiamondRecHit>> dsv_rechits;
```

Comment:
Maybe const here too?

Reply:
I don't think `dsv_rechits` can be const. It is assigned in `searchForProduct`.